#!/usr/bin/env python
PACKAGE = "image_rotate"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("angle", double_t, 0, "image rotation angle", -10, -90, 90)
gen.add("shift_a", int_t, 0, "image shift up upper", 10, 0, 200)
gen.add("shift_b", int_t, 0, "image shift up lower", 10, 0, 200)
gen.add("epi_1", int_t, 0, "epipolar one", 90, 0, 700)
gen.add("epi_2", int_t, 0, "epipolar two", 281, 0, 700)
gen.add("epi_3", int_t, 0, "epipolar three", 470, 0, 700)

exit(gen.generate(PACKAGE, "image_rotate", "rotate_node"))